modules/add_credit/tests/src/Functional/AddCreditFunctionalTestBase.php

```diff
@@ -55,6 +55,7 @@ class AddCreditFunctionalTestBase extends MonetizationFunctionalTestBase {
     'commerce_payment_test',
     'commerce_store',
     'commerce',
+    'node',
```

Comment:
Was this added to mitigate the twig issue introduced in 1.38.x?

We have a fix for that in https://github.com/apigee/apigee-m10n-drupal/pull/62/commits/7cb776ef92a9a37196db90621d575d54635a3b0f